packages/clerk-js/src/core/clerk.ts

```diff
@@ -835,6 +837,35 @@ export class Clerk implements ClerkInterface {
       beforeUnloadTracker?.stopTracking();
     }
```

Comment:
Now that `beforeEmit` is deprecated, we can also display a warning at runtime. We can some utilities you can use [here](https://github.com/clerk/javascript/blob/1c7e105a32fd492cc175ef9fd1c1fa0428c259dc/packages/shared/src/deprecated.ts#L3)

Reply:
Ah thanks for pointing that out! I've added the warning as suggested ✅